extensions/compose/src/extension.ts

```diff
@@ -89,12 +80,6 @@ export async function activate(extensionContext: extensionApi.ExtensionContext):
           extensionApi.context.setValue('composeDownloadVersion', composeVersionMetadata.tag, 'onboarding');
         }
       }
-
-      // Log if it's downloaded and what version is being selected for download (can be either latest, or chosen by user)
-      telemetryLogger.logUsage('compose.onboarding.checkComposeDownloaded', {
-        downloaded: isDownloaded === '' ? false : true,
-        version: composeVersionMetadata?.tag,
-      });
```

Comment:
should we remove the current events being done ?

I mean, it's interesting to know the workflow of users but for a detailed event it might be good to know more specifics like the version of compose being installed by users ?

Reply:
@deboer-tim WDYT?

Reply:
We will keep both